extensions/wikia/AdEngine/src/bidders.js

```diff
@@ -405,7 +405,10 @@ export function getBiddersContext(skin) {
 						enabled: false,
 						slots: {
 							FEATURED: {},
```

Comment:
What about `FEATURED`?

Reply:
When I add the properties we've added I can notice the bid goes on `SyntheticTests/Premium/FeaturedVideo?wikia_video_adapter=2000&&pbjs_debug=1` but there is no cow ad - as we discussed we probably need to change targeting of blue ad :)